res/css/style.css

```diff
@@ -183,15 +183,14 @@ body, #root, .profileViewer {
   background-color: #F5F5F5;
 }
 .treeViewRow.selected {
-  background-color: -moz-dialog;
+  background-color: var(--grey-30);
   color: black;
 }
 .treeViewRow.dim > .treeViewMainColumn {
   opacity: 0.7
 }
 .treeViewBody:focus .treeViewRow.selected {
-  background-color: highlight;
-  color: highlighttext;
+  background-color: var(--grey-30);
```

Comment:
Hey @zoepage, do you think you'll be able to handle Greg's feedback here (that is: reverting this change) so that we can land this ? :) Thanks !